metadata.rb

```diff
@@ -17,15 +13,15 @@
 supports 'windows'
 
 # build-essential is obsolete in chef 14+
-depends 'build-essential' # ~FC121
+# ~FC121
```

Comment:
Should be able to drop this food critic comment, and the build essential comment on line above